Dvorak-Right-Handed/KeyboardView.swift

```diff
@@ -6,13 +6,15 @@
 // used this tutorial to make xib: https://iostutorialjunction.com/2020/04/how-to-create-custom-uiview-class-with-xib-in-swift-tutorial.html
 //  Copyright © 2021 Taylor Shaw. All rights reserved.
 //
+/* This file contains the KeyboardView class, which owns both keyboard xib files.
```

Comment:
Consider using double slash comments to match the above comments.

Reply:
done